shared/uavobjectdefinition/hwaq32.xml

```diff
@@ -123,7 +123,7 @@
 		<field name="USB_HIDPort" units="function" type="enum" elements="1" options="USBTelemetry,Disabled" defaultvalue="USBTelemetry">
 			<description/>
 		</field>
-		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" defaultvalue="Disabled">
+		<field name="USB_VCPPort" units="function" type="enum" elements="1" parent="HwShared.USB_VCPPort" defaultvalue="Disabled">
```

Comment:
It may be possible with further discussion to pull the "Disabled" off these but I didn't feel like advocating for it with this PR (it will consume RAM but be nice in various ways).